frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -721,9 +721,13 @@ function ReaderHighlight:onHoldRelease()
                     },
                     {
                         text = _("Add Note"),
-                        enabled = false,
+                        enabled = true,
                         callback = function()
-                            self:addNote()
```

Comment:
This leaves an (even more) orphaned function around. It's probably cleaner to write the below in that function instead.

Reply:
This function currently looks like this:
```lua
function ReaderHighlight:addNote()
    self:handleEvent(Event:new("addNote"))
    logger.dbg("add Note")
end
```
`ReaderHighlight:editHighlight` doesn't emit any event. Should I add "addNote" event there, or should I remove it? I'm not sure if there are any handlers for it, I'll check it when I get access to my VM later to make sure.

Reply:
The statistics plugin responds to it.

Reply:
When editing you're not adding anything, so I don't quite see why you'd want to add an event there?

Reply:
So, this one should be fixed to be `self.ui:handleEvent(Event:new("AddNote"))`, as there is a `ReaderStatistics:onAddNote()`
(But it's just a counter, and there is no onDelNote - but I guess that doesn't really matter as notes are just highlights.)

(`self:handleEvent(Event:new("addNote")`) is currently a no-op, as there is no `ReaderHighlight:onaddNote` - which wouldn't make any sense anyway).

I thought I would agree of letting the callback as it is, and moving that bit of code in the `ReaderHighlight:addNote()` - so we have more room there if we want to do more.
But I see most of the other callbacks that do some UIManager:show() have that code in the callback itself - other ReaderHighlight methods used by callbacks are mostly non- UIManager stuff.
So, I guess I wouldn't mind either way.

(just remove the `enabled = true` as it's the default when unspecified).